data/hai/unfettered 0 prologue.txt

```diff
@@ -1473,7 +1473,7 @@ mission "<uhai> 4: Scan Geocoris"
 	on decline
 		event "unfettered: analyzed the scans" 30 60
 	on abort
-		dialog `Mera hails you when she learns of you aborting this mission. "This will slow us down significantly, by a month at least."`
+		dialog `Mera hails you when she learns of you aborting this mission. "How dare you abort?! Now I'll have to find another way to get those scans! This will slow us down significantly, by a month at least."`
```

Comment:
She sounds less angry/excited that way though. Maybe
```suggestion
		dialog `Mera hails you when she learns of you aborting this mission. "How dare you abort?! Now I'll have to find another way to get those scans! This will slow us down significantly, by a month at least!"`
```

Reply:
it's fine, she's more disappointed when she says that (but yeah she often is excited haha)